src/pipelines/rust/output.rs

```diff
@@ -31,6 +32,8 @@ pub struct RustAppOutput {
     pub import_bindings_name: Option<String>,
     /// The target of the initializer module
     pub initializer: Option<String>,
+    /// The version of wasm-bindgen used
+    pub wasm_bindgen_version: Version,
```

Comment:
Instead of passing on the version, I'd prefer to have a flag, indicating what should be done. e.g. `wasm_bindgen_object`. Unless there are good reasons not to.